docs/data/material/customization/css-theme-variables/overview.md

```diff
@@ -11,9 +11,15 @@ If this is your first time encountering CSS variables, you should check out [the
 
 ## Introduction
 
-CSS theme variable support is a new feature in Material UI added in [`v5.6.0`](https://github.com/mui/material-ui/releases/tag/v5.6.0) (but not enabled by default). It tells Material UI components to use the generated CSS theme variables instead of raw values. This provides significant improvements in developer experience related to theming and customization.
+CSS theme variable tells Material UI components to use the generated CSS theme variables instead of raw values. This provides significant improvements in developer experience related to theming and customization.
```

Comment:
A few thoughts about this particular sentence:
- Maybe we should add how that's better for theming/customization? 
- "raw values" felt vague; maybe it could be "CSS theme variables replace raw styling values in..." or similar?
- In the very last part: "when theming and customizing" could we reference "components" back with "when theming and customizing _them_"? Maybe it's grammatically correct already, but it sounded weird to me at first read 😬

Reply:
I think "when theming and customizing" can be removed.

Reply:
```suggestion
CSS theme variables replace raw values in Material UI components for a better developer experience because in the browser dev tool, you will see which theme token is used as a value.
```